CMSgov/mdct-mcr
# Disable forms for admins

## Description

Disable report fields for admin users

### How to test

log in as an admin and see that the report fields are disabled (including nested fields), and report fields for state users are not disabled.

### Changed Dependencies


## Code author checklist

## Reviewer checklist (two different people)
